socorro/processor/processor_app.py

```diff
@@ -353,7 +351,7 @@ def _setup_source_and_destination(self):
 
         self.processor = self.config.processor.processor_class(
             self.config.processor,
-            self.quit_check
+            quit_check_callback=self.quit_check
```

Comment:
Without this fix, the `self.quit_check` becomes the value of `rule_set` which prevents the processor from starting.